### ISSUE TITLE
FORTRAN wrapper for CoolProp

### ISSUE THREAD

**Issue by ibell** [ibell/coolprop#67]

**Comment by jowr** [comment]
My colleague Brian is working on that.

**Comment by ibell** [comment]
Even better :)

**Comment by ibell** [comment]
@jowr Any update on the FORTRAN wrapper?

**Comment by ibell** [comment]
+1 , just got another request.

**Comment by jowr** [comment]
OK, let's not wait any longer. I will have a look at it myself...

**Comment by jowr** [comment]
We need a C wrapper to be called from Fortran. Does anyone have such a file already?

**Comment by jowr** [comment]
I added a basic file that shows the way to go. It is tested on Linux and it works well, it does NOT work on MacOS and I could not test on Windows. It would be great to get some help here because I am all but a Fortran expert... I also added a rudimentary C interface.

**Comment by ibell** [comment]
Cool!  What's the issue on OSX?  I can test on windows I hope. Whats the
build procedure?

**Comment by jowr** [comment]
There is a `build.sh` in both the `C` and the `Fortran` subdirectory. Have a look there for build order etc.

**Comment by ibell** [comment]
I can't get the executable to build on windows.  I made a batch script with
contents

```
gfortran -g -c examplef.for
gcc -g -c ../C/standaloneC.c
gfortran -g -o example examplef.o standaloneC.o -lCoolProp -ldl -lgfortran
```

which then dies on the third line with

```
C:\Users\Belli\Documents\Code\CoolProp\wrappers\Fortran>gfortran -g -o
example examplef.o standaloneC.o -lCoolProp -ldl -lgfortran
c:/mingw32-xy/bin/../lib/gcc/mingw32/4.5.2/../../../../mingw32/bin/ld.exe:
cannot find -lCoolProp
c:/mingw32-xy/bin/../lib/gcc/mingw32/4.5.2/../../../../mingw32/bin/ld.exe:
cannot find -ldl
```

If I am not mistaken `-ldl` is to allow for the use of dynamic libraries,
right?  Isn't it built-in? What should I do about the `-lCoolProp` flag
on windows?  Do I need to specify the dll search path somewhere?

**Comment by jowr** [comment]
I'll talk to my Windows-using colleagues, please stay on hold for a little while.

**Comment by jowr** [comment]
Fixed in ea2028ea5fb9a8dfb4278eeb5a693e8cf049a54c . We can now use CoolProp with our ancient Fortran77 software on Windows and Linux... 
Please reopen in case anyone needs a more sophisticated wrapper.
